src/site/content/en/handbook/quick-start/index.md

```diff
@@ -11,9 +11,9 @@ description: |
 Content creation for web.dev has three phases: planning, writing, and publishing.
 
 ## Planning
-1. Start a content proposal issue on GitHub using [this issue template](https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=content+request&template=content_request.md&title=). The checklist in your issue will guide you through the rest of the process.
+1. Start a content proposal issue on GitHub using [this issue template](https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=content+request&template=content_request.md&title=). The checklist in your issue guides you through the rest of the process.
```

Comment:
This instance of present tense feels very weird to me. You haven't read the doc yet, so it can't already be guiding yet. I know the canonical advice about the eternal present, but I think it leads to less intuitive language in some cases. Am I crazy?